src/utils/file.ts

```
@@ -296,3 +296,11 @@
 export const hashSHA256 = (text: string) => {
   return createHash("sha256").update(text, "utf8").digest("hex");
 };
+
+export const isFile = (path: string): boolean => {
```

ESLint

[error] 'path' is already declared in the upper scope on line 2 column 8.

(no-shadow)